docs/source/asr/speaker_diarization/datasets.rst

```diff
@@ -1,72 +1,133 @@
-  - diarizer.oracle_num_speakers = null (performing unknown speaker case)
-  - diarizer.speaker_embeddings.model_path = ``ecapa_tdnn`` (This model is trained on voxceleb dataset. ``Use this model for simialr non-telephonic speech datasets``)
-  - diarizer.speaker_embeddings.window_length_in_sec = 1.5
-  - diarizer.speaker_embeddings.shift_length_in_sec = 0.75 
+AMi Meeting Corpus
+~~~~~~~~~~~~~~~~~~
 
-Input paths2audio_files, paths2rttm_files and oracle_vad_manifest by following steps as shown above
+The following are the suggested parameters for reproducing the diarization performance for AMI test set.
 
-CallHome LDC97S42 (CH109)
--------------------------
+.. code-block:: bash
+
+  diarizer.manifest_filepath="/path/to/AMItest_input_manifest.json"
+  diarizer.oracle_num_speakers=null # Performing unknown speaker case
+  diarizer.oracle_vad=True # Use oracle VAD extracted from RTTM files.
+  diarizer.collar=0.25
+  diarizer.ignore_overlap=True 
+  diarizer.speaker_embeddings.model_path = ``titanet_large`` 
+  diarizer.speaker_embeddings.window_length_in_sec=[3,1.5,1.0,0.5] # Multiscale setting
+  diarizer.speaker_embeddings.shift_length_in_sec=[1.5,0.75,0.5,0.25] # Multiscale setting 
+  diarizer.speaker_embeddings.parameters.multiscale_weights=[0.4,0.3,0.2,0.1] # More weights on the longer scales
+
+This setup is expected to reproduce speaker error rate  of 1.17% on AMI test set.
+
+To evaluate the performance on AMI Meeting Corpus, the following instructions can help.
+  - Download AMI Meeting Corpus from `AMI website <https://groups.inf.ed.ac.uk/ami/corpus/>`_.
+  - Download the test set (whitelist) from `Pyannotate AMI test set whitelist <https://raw.githubusercontent.com/pyannote/pyannote-audio/master/tutorials/data_preparation/AMI/MixHeadset.test.lst>`_.
+  - The merged RTTM file for AMI test set can be downloaded from `Pyannotate AMI test set RTTM file <https://raw.githubusercontent.com/pyannote/pyannote-audio/master/tutorials/data_preparation/AMI/MixHeadset.test.rttm>`_. Note that this file should be split into individual rttm files. Download split rttm files for AMI test set from `AMI test set split RTTM files <https://raw.githubusercontent.com/tango4j/diarization_annotation/main/AMI_corpus/test/split_rttms.tar.gz>`_.
+  - Generate an input manifest file using ``<NeMo_git_root>/scripts/speaker_tasks/pathsfiles_to_manifest.py``
+
+
+CallHome American English Speech (CHAES), LDC97S42: 2-speaker subset (CH109)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CH109 is a subset of the CHAES dataset which has only two speakers in one session. 
+The following are the suggested parameters for reproducing the diarization performance for the CH109 set.
+
+.. code-block:: bash
+
+  diarizer.manifest_filepath="/path/to/ch109_input_manifest.json"
+  diarizer.oracle_num_speakers=2 (Since there are exactly 2 speakers per each CH109 session)
```

Comment:
This is not needed here right? user would prepare it with pathsfiles_to_manifest.py

Reply:
I think it is better to be here to make sure they use oracle number of speakers with the number 2.